test/processors/base_processor_test.rb

```diff
@@ -23,6 +21,42 @@ class BaseProcessorTest < ActiveSupport::TestCase
     }
   }
 
+  let(:audio_monster) do
+    Minitest::Mock.new
+  end
+
+  let(:processor) do
+    BaseProcessor.new(logger: Logger.new('/dev/null')).tap do |p|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.